app/models/track.rb

```diff
@@ -16,7 +16,7 @@ class Track < ApplicationRecord
     :ignore_regexp,
     to: :git
 
-  delegate :head_sha, to: :git, prefix: :git
+  delegate :head_sha, to: :git, prefix: true
```

Comment:
This is implicit, not explicit now 🙂 

I preferred explicit

Reply:
My logic is that if we rename the `git` method in `Track` to `repo` (for example), we don't want all the things that use `git_head_sha` to fail because this method gets renamed to `repo_head_sha`.